### PR TITLE
fix(service-portal): UI fixes

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

Various UI fixes for the rights portal

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by saevarma — APPROVED** [review]
Core files LGTM

File: libs/service-portal/health/src/screens/Dentists/Dentists.tsx [hunk 1]
```diff
@@ -44,26 +45,20 @@ const Dentists = () => {
         dateTo: selectedDateTo,
       },
     },
-    fetchPolicy: 'no-cache',
```
Comment (thordurhhh) [inline libs/service-portal/health/src/screens/Dentists/Dentists.tsx:47]: I believe this was in place to force fetch an update on redirect.

**Review by thordurhhh — CHANGES_REQUESTED** [review]
One minor comment, 🙌

**Review by thordurhhh — APPROVED** [review]
nice 🐬

**Review by Tryggvig — APPROVED** [review]
LGTM for ui

**Pull request merged**